spec/integration/spec_file_load_errors_spec.rb

```diff
@@ -54,6 +54,37 @@
     EOS
   end
 
+  it 'prints a single error when it happens on --require files' do
+    write_file_formatted "helper_with_error.rb", "raise 'boom'; class Perry; end"
+
+    write_file_formatted "1_spec.rb", "
+      RSpec.describe Perry do
```

Comment:
We don't recommend classes in describe, so a string is better here, e.g. `"a file that will never load"` would be appropriate here.

Reply:
This is related to the other comment. I'm trying to simulate the case when loading of the spec files is dependent on successful loading of the helper.

Reply:
This is only out of curiosity since we're at it. So the RSpec core team disagrees with https://github.com/rubocop-hq/rubocop-rspec/blob/4ab2e3f6654feb74fb6ccadf9403c64a2b50e8c1/lib/rubocop/cop/rspec/describe_class.rb? Will you deprecate specifying classes in `describe` in the future? What's the rationale behind recommending against it? Thanks! :heart:

Reply:
> So the RSpec core team disagrees with <rubocop rule link>

Absolutely, Rubocop isn't best practise, it's set of tool to enforce opinions

> Will you deprecate specifying classes in describe in the future? 

Maybe.

> What's the rationale behind recommending against it?

The problem is that using a constant to describe a class forces you to have loaded the class in advance, in the case of Rails and other frameworks this is expensive, especially if you're going to skip the spec, we want to load all the specs without needing anything, and then boot expensive things when we know we're going to need them.

Reply:
I just tried creating a new rails application, installing `rspec` on it and generating some scaffold and I observe that:

* Rails is booted inside the default `rails_helper.rb`
* Classes instead strings are used in both controller and model specs by default.

This seems to contradict your recommendation.

Reply:
We also require the rails_helper by default, in those generators, this is not RSpec Rails though. Here there is no magic autoload.

Reply:
I was just mentioning it since you brought up Rails as an example where constants are not recommended for describe. Anyways, I took this off-topic from the current PR, sorry :pray:

Reply:
How about this:
```
RSpec.describe "A broken spec file that will raise when loaded" do
  # this is to trigger an error during spec loaded
  raise "boom"
end
```

Reply:
Ok, I'll try to make the spec look the way you like it! :+1: